docs/tutorials/1 - Introductory Example - Fixed Tilt simple setup.py

```diff
@@ -39,7 +39,12 @@
 
 
 import os
-testfolder = os.path.abspath(r'..\..\bifacial_radiance\TEMP')  
+from pathlib import Path
+
+testfolder = Path().resolve().parent.parent / 'bifacial_radiance' / 'TEMP'
+
+# Another option using relative address; for some operative systems you might need '/' instead of '\'
+# testfolder = os.path.abspath(r'..\..\bifacial_radiance\TEMP')  
 
 print ("Your simulation will be stored in %s" % testfolder)
 
```

Comment:
@shirubana - in general do we want to distribute .py tutorial files along with the .ipynb files?  There's an old .py version of tutorial 1 hanging around in here.  We can add the .py files in this directory to .gitignore if we don't need to distribute them..

Reply:
yes -- advanced users might want to do the .py to look at variables actively in their spyder 'variable' tab menu. Also, I've had instances where journals die on me (I'm mean to them or something, I don't know), and the .py have saved my life.


![image](https://user-images.githubusercontent.com/22824806/70671021-f8225500-1c37-11ea-9d23-0ba68d990ea1.png)
Not.